stickyuploads/storage.py

```diff
@@ -9,7 +10,16 @@ class TempFileSystemStorage(FileSystemStorage):
 
     def __init__(self):
         super(TempFileSystemStorage, self).__init__(
-            location=tempfile.gettempdir(), base_url=None)
+            location=self.get_dir(), base_url=None)
+
+    def get_dir(self):
+        """Return operational directory, create it if it doesn't exists"""
+        if hasattr(settings, 'BASE_DIR'):
```

Comment:
Django's `startproject` already uses a `BASE_DIR` setting for building paths relative to the project directory.